src/server/enrollment-system.js

```diff
@@ -1094,13 +1092,11 @@ function veteranToAssociationCollection(veteran) {
   const spouse = spouseToAssociation(veteran);
   associations.concat(children);
   if (spouse) { associations.push(spouse); }
-
   if (_.isEmpty(associations)) {
     return undefined;
   }
-  const result = associations.map((association) => { return { association }; });
-  console.log(result);
-  return result;
+  const result = associations.length === 1 ? associations.pop() : associations;
+  return { association: result };
```

Comment:
@robbiethegeek after updating the conformance test XML, I changed this to just include `associations` as-is, and the results came out the same for `no-children.xml` which only has one `association` record.